pipelines/stack_os_matrix.groovy

```diff
@@ -41,6 +41,10 @@ notify.wrap {
     buildParams['LSST_SPLENV_REF'] = params.SPLENV_REF
   }
 
+  if (params.SAVE_CACHE) {
```

Comment:
This feels a bit wrong somehow (the build params and env vars don't seem to be interacting properly), but I can't deny that it should work.  Remember this for a refactoring (or at least documenting) of how the pipeline interfaces work.